modules/service/src/main/scala/lucuma/odb/service/ObservationService.scala

```diff
@@ -428,6 +447,128 @@ object ObservationService {
           _             <- ResultT(asterismService.setAsterism(pid, NonEmptyList.of(newOid), input.asterism))
         yield CloneIds(origOid, newOid)).value
         
+      // ***********************
```

Comment:
I toyed with the idea of creating a validation service instead of adding it to the observation service. I decided to go this way, but could be convinced to move it.